cluster-up/cluster/kind/common.sh

```diff
@@ -102,19 +102,27 @@ function prepare_workers() {
     done
 }
 
+function _get_nodes() {
+    _kubectl get nodes --no-headers
+}
+
+function _get_pods() {
+    _kubectl get pods --all-namespaces --no-headers
+}
+
 function _fix_node_labels() {
     # Due to inconsistent labels and taints state in multi-nodes clusters,
     # it is nessecery to remove taint NoSchedule and set role labels manualy:
     #   Master nodes might lack 'scheduable=true' label and have NoScheduable taint.
     #   Worker nodes might lack worker role label.
-    master_nodes=$(_kubectl get nodes --no-headers | grep -i $MASTER_NODES_PATTERN | awk '{print $1}')
+    master_nodes=$(_get_nodes | grep -i $MASTER_NODES_PATTERN | awk '{print $1}')
```

Comment:
Though not directly related to you change, since you already move stuff into functions, I think that we have an opportunity to improve here:

I think that the `node-role.kubernetes.io/master` label can tell you exactly who the masters are. Also, you don't need to use `awk`, you can use `kubectl's output formatter instead... maybe something like:

```
function _get_master_names() {
	_kubectl get nodes \
		--no-headers \
		-l node-role.kubernetes.io/master="" \
		-o custom-columns=:.metadata.name
}
```

Since this is not directly related, feel free not to do this change

Reply:
Hi Daniel, we search for worker nodes by name because there seem to be an issue with the worker labeling in KIND with multi-nodes clusters, this is why I created PR this in the first place.

Reply:
Ah I see... this is why an informative commit message is important :smile:

Reply:
@danielBelenky yep, totally my fault, I am reworking a little this so change is smaller, just don't relabel nodes if not needed.